### PR TITLE
[Enhance](ComputeNode) K8sDeployManager support computeNode

### PR DESCRIPTION
**Pull request by zddr**
# Proposed changes

Issue Number: close #xxx

## Problem summary

Describe your changes.

1.allow have no ELECTABLE or BACKEND

2.add cn NodeType

3.delete deprecated code 

## Checklist(Required)

* [x] Does it affect the original behavior
* [ ] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [x] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 34.9 seconds
 stream load tsv:          479 seconds loaded 74807831229 Bytes, about 148 MB/s
 stream load json:         37 seconds loaded 2358488459 Bytes, about 60 MB/s
 stream load orc:          69 seconds loaded 1101869774 Bytes, about 15 MB/s
 stream load parquet:          29 seconds loaded 861443392 Bytes, about 28 MB/s
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230215133325_clickbench_pr_97081.html

File: fe/fe-core/src/main/java/org/apache/doris/deploy/DeployManager.java [hunk 1]
```diff
@@ -148,38 +146,51 @@ public DeployManager(Env env, long intervalMs) {
...
-                 envBackendServiceGroup, envBrokerServiceGroup);
+        LOG.info("get deploy env: {}, {}, {}, {}, {}", envElectableFeServiceGroup, envObserverFeServiceGroup,
+                envBackendServiceGroup, envBrokerServiceGroup, envCnServiceGroup);
 
-        // electableFeServiceGroup and backendServiceGroup must exist
-        if (Strings.isNullOrEmpty(electableFeServiceGroup) || Strings.isNullOrEmpty(backendServiceGroup)) {
-            LOG.warn("failed to init service group name."
-                    + " electableFeServiceGroup: {}, backendServiceGroup: {}",
-                     electableFeServiceGroup, backendServiceGroup);
-            System.exit(-1);
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/deploy/DeployManager.java:166]: I don't remember why we exit if service group does not exist.
But are you sure it is OK to remove it?

Comment (zddr) [reply fe/fe-core/src/main/java/org/apache/doris/deploy/DeployManager.java:166]: This is to be compatible with only cn running on k8s

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**